test/io/jesi/customs/strict/kaocha_test.clj

```diff
@@ -0,0 +1,7 @@
+(ns io.jesi.customs.strict.kaocha-test
+  (:refer-clojure :exclude [=])
+  (:require
+    [io.jesi.customs.strict :refer :all]))
+
+(deftest ^:kaocha/skip kaocha-diff-test
+  (is= {:a 1} {:a 2}))
```

Comment:
This is confusing. Is this just to check it works? Should we add a comment as to why it's skipped or actually `not=`

Reply:
Yeah I couldn't figure out how to to intercept kaocha reporting so had to have a failing test to test the diff works. I'll add a commment